WordPressKit/BlogServiceRemoteREST.m

```diff
@@ -95,6 +95,62 @@ - (void)getAuthorsWithSuccess:(UsersHandler)success
                           }];
 }
 
+- (void)getAllAuthorsWithNumber:(NSNumber *)number
+                        success:(UsersHandler)success
+                        failure:(void (^)(NSError *error))failure
+{
+    NSMutableArray *remoteUsers = [NSMutableArray array];
+    [self getAllAuthorsWithRemoteUsers:remoteUsers
+                                number:number
+                                offset:nil
+                               success:success
+                               failure:failure];
+}
+
+- (void)getAllAuthorsWithRemoteUsers:(NSMutableArray <RemoteUser *>*)remoteusers
+                                number:(NSNumber *)number
+                                offset:(NSNumber *)offset
+                               success:(UsersHandler)success
+                               failure:(void (^)(NSError *error))failure
+{
```

Comment:
Rather than need to define and pass `remoteUsers` in the calling method, you might consider checking for `remoteUsers==nil` in this method and instantiating a copy if one is needed.  It would keep the logic a bit more encapsulated.